.env

```diff
@@ -0,0 +1,4 @@
+NOTARY_CONTRACT_NAME=Notary
+API_ENDPOINT=http://localhost:8080
+VCHAIN=42
+ORBS_TOPOLOGY={"nodes":[{"name": "local", "host":"localhost:8080"}]}
```

Comment:
I suggest you add something faulty like `localhost:8989` so we can see how the errors are handled (timeouts are respected, etc).